fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -826,49 +838,100 @@ class HomeFragment : Fragment() {
+                    dismissButtonColor = getColor(context, R.color.fx_mobile_icon_color_oncolor),
+                    indicatorDirection = CFRPopup.IndicatorDirection.UP,
+                ),
+                onDismiss = {
                     PrivateBrowsingShortcutCfr.cancel.record()
-                    privateBrowsingRecommend.dismiss()
-                }
-            }
-            // We want to show the popup only after privateBrowsingButton is available.
-            // Otherwise, we will encounter an activity token error.
-            binding.privateBrowsingButton.post {
-                runIfFragmentIsAttached {
                     context.settings().showedPrivateModeContextualFeatureRecommender = true
                     context.settings().lastCfrShownTimeInMillis = System.currentTimeMillis()
-                    privateBrowsingRecommend.showAsDropDown(
-                        binding.privateBrowsingButton,
-                        0,
-                        CFR_Y_OFFSET,
-                        Gravity.TOP or Gravity.END,
-                    )
-                }
+                    recommendPrivateBrowsingCFR?.dismiss()
+                },
+                text = {
+                    FirefoxTheme {
+                        Text(
+                            text = context.getString(R.string.private_mode_cfr_message_2),
+                            color = FirefoxTheme.colors.textOnColorPrimary,
+                            style = FirefoxTheme.typography.headline7,
+                            modifier = Modifier
+                                .semantics {
+                                    testTagsAsResourceId = true
+                                    testTag = "private.message"
+                                },
+                        )
+                    }
+                },
+                action = {
+                    FirefoxTheme {
+                        Button(
+                            onClick = {
+                                PrivateBrowsingShortcutCfr.addShortcut.record(NoExtras())
+                                PrivateShortcutCreateManager.createPrivateShortcut(context)
+                                context.settings().showedPrivateModeContextualFeatureRecommender = true
+                                context.settings().lastCfrShownTimeInMillis = System.currentTimeMillis()
+                                recommendPrivateBrowsingCFR?.dismiss()
+                            },
+                            colors = ButtonDefaults.buttonColors(backgroundColor = PhotonColors.LightGrey30),
+                            modifier = Modifier
+                                .padding(top = 16.dp)
+                                .heightIn(36.dp)
+                                .fillMaxWidth()
+                                .semantics {
+                                    testTagsAsResourceId = true
+                                    testTag = "private.add"
+                                },
```

Comment:
For better consistency with the popup, we could also specify the shape of this button as `RoundedCornerShape(8.dp)`, as specified in `rounded_gray_corners.xml`